### ISSUE TITLE
Change default JSON encoding of `Data`

### ISSUE THREAD

**Issue by kamalmarhubi** [sandstorm-io/capnproto#259]
The current encoding as an array of byte values is verbose and ugly and probably never what someone wants. If there's to be a default, base64 encoding is probably better. Or require the user to explicitly add a handler, though that would likely make an annoying user experience.

**Comment by kentonv** [comment]
The reason I went with array-of-ints is because that will by default produce an object in Javascript which behaves much like Data (an array of bytes). Array-of-ints also makes a reasonable amount of sense if your data blobs are short and sparse.

I agree that base64 is *usually* what people actually want, but requires explicit support on the receiving end. Note that IE only added built-in support for base64 decoding in version 10. Also there are a lot of variants of base64, e.g. padding or no padding, different choices of digits 62 and 63, etc. So it's all fairly muddled.

I would be in favor of providing a readymade base64 handler to make it as easy as possible for people to use base64 if that's what they want.

**Comment by kamalmarhubi** [comment]
Makes sense. I'll update this issue title to make it a feature request for a base64 handler.
